sale_edi.py

```diff
@@ -758,9 +762,17 @@ def import_sales(cls, edi_sale=None):
         for fname in files:
             if fname[-4:].lower() not in KNOWN_EXTENSIONS:
                 continue
-            with open(fname, 'r', encoding='latin-1') as fp:
-                data = fp.readlines()
-                sale_edi = cls.import_edi_file([], data)
+            for count in range(retry, -1, -1):
+                if count != retry:
+                    time.sleep(0.02 * (retry - count))
+                try:
+                    with open(fname, 'r', encoding='latin-1') as fp:
+                        data = fp.readlines()
+                        sale_edi = cls.import_edi_file([], data)
+                except Exception:
+                    logger.error('fail to import edi file', exc_info=True)
+                    return
```

Comment:
I don't think we can ignore such an important error.

Reply:
Sometimes mount is not available for few "seconds".  For this reason, retry some times to connect.